src/fastmcp/server/middleware/response_limiting.py

```
@@ -0,0 +1,400 @@
+            )
+
+        # Check if tool uses x-fastmcp-wrap-result by inspecting output_schema
+        is_wrapped = False
+        if context.fastmcp_context is not None:
+            try:
+                tool = await context.fastmcp_context.fastmcp.get_tool(tool_name)
+                if tool and tool.output_schema:
+                    is_wrapped = self._is_wrapped_simple_value(tool.output_schema)
+            except Exception:
+                # If we can't get the tool info, fall back to conservative behavior
+                pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)